modules/src/ics04_channel/msgs.rs

```diff
@@ -468,7 +468,7 @@ impl MsgTimeout {
         packet: Packet,
         next_sequence_recv: Option<u64>,
         proof: CommitmentProof,
-        proof_height: Height,
+        proof_height: RawHeight,
```

Comment:
Minor comment: it's a bit odd to see raw type side by side with domain type. I guess we could have just as well used `crate::Height` instead of `ibc_proto::ibc::core::client::v1::Height as RawHeight;` unless I'm missing something.

We can fix this in a later iteration, and perhaps also split the modules/src/ics04_channel/msgs.rs file like we did in ICS3.

Reply:
Agreed, let’s use `crate::Height` instead.